cg/meta/workflow/taxprofiler.py

```diff
@@ -158,3 +161,11 @@ def parse_multiqc_json_for_sample(sample_name: str, multiqc_json: list[dict]) ->
                     metrics_values.update(sample_values)
 
         return metrics_values
+
+    @staticmethod
+    def get_deliverables_template_content() -> list[dict]:
```

Comment:
Am I correct? in any case it would be useful to specify the types inside the dict
```suggestion
    def get_deliverables_template_content() -> list[dict[str, str]:
```

Reply:
Added now.